src/Lykke.Service.Subscribers.Services/ShutdownManager.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Threading.Tasks;
+using Common.Log;
+using Lykke.Service.Subscribers.Core.Services;
+
+namespace Lykke.Service.Subscribers.Services
+{
+    // NOTE: Sometimes, shutdown process should be expressed explicitly. 
+    // If this is your case, use this class to manage shutdown.
+    // For example, sometimes some state should be saved only after all incoming message processing and 
+    // all periodical handler was stopped, and so on.
+    
+    public class ShutdownManager : IShutdownManager
```

Comment:
I would delete this class (as well as StartupManager) since there is no logic here, but just a stub.

Reply:
What about this comment?

Reply:
If we don't need this class I will remove the Startup method "StopApplication", isn't it necessary?

Reply:
Sure.